src/community/edge_betweenness.c

```diff
@@ -319,18 +319,26 @@ igraph_error_t igraph_community_eb_get_merges(const igraph_t *graph,
     return IGRAPH_SUCCESS;
 }
 
-/* Find the smallest active element in the vector */
-static igraph_integer_t igraph_i_vector_which_max_not_null(const igraph_vector_t *v,
-                                                   const bool *passive) {
-    igraph_integer_t which, i = 0, size = igraph_vector_size(v);
+/* Find the index i for which v[i] / w[i] is the smallest
+ * and the corresponding element is active (i.e. !passive[i]).
+ * If w is a null pointer then all w[i] are assumed to be 1.0.
+ * Assumes that at least one element is active. */
+static igraph_integer_t igraph_i_vector_which_max_not_null(
```

Comment:
Should be renamed to `igraph_i_vector_pair_find_max_ratio` or something like that as we now have two vectors and we are searching for the maximum in the ratio of them.